plugin/govim.vim

```diff
@@ -16,6 +16,11 @@ if !has("patch-8.1.1711")
   finish
 endif
 
+if has("patch-8.2.0425") && !has("patch-8.2.0466")
+  echoerr "Vim versions v8.2.0425 <= N < v8.2.0466 have a bug that affects govim. Please update to another version"
```

Comment:
Isn't it `0452`?
```suggestion
if has("patch-8.2.0452") && !has("patch-8.2.0466")
  echoerr "Vim versions v8.2.0452 <= N < v8.2.0466 have a bug that affects govim. Please update to another version"
```

Reply:
It is, thank you for that catch! I tested locally with Vim v8.2.0452... but then didn't read what I'd written closely enough!